docs/source/advanced/boosted_frame.rst

```diff
@@ -45,14 +45,13 @@ the number of required PIC iterations is reduced in the boosted frame:
 
 .. math::
 
-    N_{iterations, boosted} \approx \frac{1}{(1+\beta_{boost})^2 \gamma_{boost}^2} N_{iterations, lab}
+    N_{iterations, boosted} \approx \frac{1}{(1+\beta_b)^2 \gamma_b^2} N_{iterations, lab}
 
 which can speed up the simulation by orders of magnitude. However, the above formula is
-only an approximation (valid for :math:`\gamma_{boost} \ll \gamma_{wake}`) and the calculation of the
-correct number of timesteps in practice is explained below (see :ref:`boosted_frame_speedup`).
-Moreover, it is noteworthy that the theoretical speedup for a given simulation setup
-can be at maximum :math:`2 \gamma_{wake}^2` for :math:`\gamma_{boost} \to \infty` in the case of
-plasma wakefield acceleration. Here, :math:`\gamma_{wake}` is the Lorentz factor
+only an approximation and the calculation of the correct number of timesteps
+in practice is discussed below (see :ref:`boosted_frame_speedup`).
+Note also that, in theory, the optimal value of :math:`\gamma_b` is
+close to :math:`\gamma_{wake}`, i.e. the Lorentz factor
```

Comment:
Some simplification in order to reduce the cognitive load on a user reading this. 
(There is already a lot of information in this page of the documentation!)

Reply:
:white_check_mark: